.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.idea/
```

Comment:
"Have you checked if there are any inconsitencies between old and new files (otf, ttf, etc.)? " 
I noticed a few differences, but I did not analyze in detail. I think this may be related to another version of the font, or a different version of bootstrap.